src/js/api/config.js

```diff
@@ -61,9 +61,53 @@ function _adjustDefaultBwEstimate(estimate) {
     return Defaults.bandwidthEstimate;
 }
 
+function _mergeProperty(localizationObj, allOptionsObj, prop) {
+    if (prop) {
+        const propToCopy = localizationObj[prop] || allOptionsObj[prop];
+
+        if (propToCopy) {
+            localizationObj[prop] = propToCopy;
+        }
+    } else {
+        localizationObj = Object.assign({}, localizationObj, allOptionsObj);
+    }
+}
+
+function _copyToLocalization(allOptions) {
+    const { advertising, related, sharing, abouttext } = allOptions;
+    const localization = Object.assign({}, allOptions.localization);
+
+    if (advertising) {
+        _mergeProperty(localization.advertising, advertising);
+    }
+
+    if (related) {
+        if (typeof localization.related === 'string') {
+            localization.related = {
+                heading: localization.related
+            };
+
+            _mergeProperty(localization.related, related, 'autoplaymessage');
```

Comment:
this should be outside of the if block, since it is not related to `localization.related` being a string